src/marketplace/eqp/v1/rest-api.md

```diff
@@ -0,0 +1,95 @@
+---
+group: marketplace-api
+title: REST API
+---
+
+ {:.bs-callout-info}
+The **Marketplace EQP REST API** is completely separate from APIs provided with Magento 2.
+<br/>There is no corresponding SOAP API.
+
+The **Marketplace EQP API** gives you access to your **Marketplace Developer Portal** resources.
+These resources are managed by a set of REST API endpoints.
+
+## Overview of resources
+
+| Resource       | Purpose |
+|----------------|---------|
+| authentication | obtain a session token |
```

Comment:
Capitalize the first letter of each entry in the right column.

Reply:
Thank you for resolving!